rifle.py

```diff
@@ -87,6 +94,26 @@ def draw_terminal(self, screen):
     def new_word(self):
         return self.word_generator.request_word(3,3)
 
+class RifleProjectile(): 
+
+	def __init__(self, universe, current_text):
+		self.universe = universe 
+		self.input_ = current_text
+
+	def update(self,events): 
+		target_ID = getTargetID(input_)
+		if (targetID): 
+			target_position = self.universe.enemies_[target_ID].position_ 	
+			print(target_position)	#test 
+			# shoot projectile towards where enemy was at that time. 
+
+	def getTargetID(input): 
+		for enemy in self.universe_.enemies(): 
+			target_tag_word = self.target_tags_[enemy.ID_]
+			if(input == target_tag_word)
+				return ids_for_target_tags_[input]
+			else		# input does not exist 
+				return 0
```

Comment:
I'd recommend returning `None` instead of 0, because having gameobjects of ID 0 may be a possibility later. Gameobjects will never had an ID of `None` so it's probably safer.